test/regfox/responses/regfox_api_get_registration_info_parser.test.js

```diff
@@ -0,0 +1,72 @@
+import { expect } from 'chai';
+
+import { parseRegfoxGetRegistrationResponse } from '../../../src/regfox/responses/regfox_api_get_registration_info_parser.js';
+
+describe('regfox_api_get_registration_info_parser', () => {
+  describe('parseRegfoxGetRegistrationResponse', () => {
+    it('doesn\'nt crash and burn on undefined input', () => {
```

Comment:
Personal nit: When writing messages like this I avoid contractions (don't -> do not) just to avoid the backslash in the middle of the word. Something about it trainwrecks my reading comprehension every time. Not a blocker, just a nitpick.

Reply:
Totally fair! Will fix. :)